packages/blade/src/components/Box/BaseBox/types/props.types.ts

```diff
@@ -93,20 +100,22 @@ type BackgroundColorString<T extends ColorObjects> = `${T}.background.${DotNotat
   Theme['colors'][T]['background']
 >}`;
 
-type BackgroundColorType =
-  | BackgroundColorString<'feedback'>
-  | BackgroundColorString<'surface'>
-  | BackgroundColorString<'action'>
-  | (string & Record<never, never>);
-
-type VisualProps = MakeObjectResponsive<
+type BaseBoxVisualProps = MakeObjectResponsive<
   {
     borderRadius: keyof Border['radius'];
-    backgroundColor: BackgroundColorType;
+    backgroundColor:
+      | BackgroundColorString<'feedback'>
+      | BackgroundColorString<'surface'>
+      | BackgroundColorString<'action'>
+      | (string & Record<never, never>);
     lineHeight: SpacingValueType;
   } & Pick<CSSObject, 'transform'>
 >;
 
+type BoxVisualProps = MakeObjectResponsive<{
+  backgroundColor: BackgroundColorString<'surface'>;
+}>;
+
 type StyledProps = Partial<
```

Comment:
Yup we can. Will do this. 

P.S. Are we exporting this to consumers? Cause I haven't exported it from our public components. Don't see any usecase as of now 🤔

Reply:
maybe we don't expose this to consumers but having this for now will establish conventions and prevent us when we expose this and at time won't even remember this context and might lead to conflicts :D

Reply:
yup right! I have changed it `StyledPropsBlade` instead because when we are typing we will start from `StyledProps` so it will show up early in the autocomplete of autoimport.